core/src/i18n/en.yaml

```diff
@@ -244,14 +244,14 @@ cluster-roles:
 clusters:
   add:
     errors:
-      failed-to-add-cluster: Failed to add a cluster
-    title: Add a Cluster
+      failed-to-add-cluster: Failed to connect a cluster
+    title: Connect a Cluster
   buttons:
     apply-configuration: Apply Configuration
     apply-kubeconfig: Apply kubeconfig
     next-step: Next Step
```

Comment:
Also, @grego952 shouldn't `step` be lowercase? I don't want to solve the problem in this task, because that's not its purpose, but I think we should remember it for later and revisit it to be consistent 🙂

Reply:
Correct. Should be `step`